CmdStarter.Tests/TestsCommon.cs

```diff
@@ -116,30 +115,46 @@ public static void AssertIEnumerablesHaveSameElements<T>(IEnumerable<T> actual,
             }
         }
 
-        public static string PrintOption(string optionName, object expectedValue)
+        public static List<string> PrepareOption(string optionName, object expectedValue)
         {
+            var args = new List<string>();
+
             switch (Type.GetTypeCode(expectedValue.GetType()))
             {
                 case TypeCode.Boolean:
-                    return OptHandling.OPTION_PREFIX + optionName;
+                    args.Add(OptHandling.OPTION_PREFIX + optionName);
+                    break;
 
                 default:
-                    if (expectedValue.GetType().IsArray)
+                    if (expectedValue is not string && expectedValue is IEnumerable values)
                     {
-                        var optionString = string.Empty;
-                        var values = (IEnumerable)expectedValue;
                         foreach (var curValue in values)
                         {
-                            optionString += " " + OptHandling.OPTION_PREFIX + optionName + " " + curValue;
+                            var value = curValue.ToString() ?? string.Empty;
+                            if (string.IsNullOrWhiteSpace(value)) continue;
+
+                            args.Add(OptHandling.OPTION_PREFIX + optionName);
+                            args.Add(value);
                         }
-                        optionString = optionString.Trim();
-                        return optionString;
                     }
                     else
                     {
-                        return OptHandling.OPTION_PREFIX + optionName + " " + expectedValue;
+                        var value = expectedValue.ToString() ?? string.Empty;
+                        if (!string.IsNullOrWhiteSpace(value))
+                        {
+                            args.Add(OptHandling.OPTION_PREFIX + optionName);
+                            args.Add(value);
+                        }
                     }
+                    break;
             }
```

Comment:
Isn't the switch case unjustified here?

Switch cases are less readable than if statements for just two elements.

Reply:
Well, it is a switch to support other types in the future. I know, for now, it is a bit unjustified.

Reply:
Oh, okay!